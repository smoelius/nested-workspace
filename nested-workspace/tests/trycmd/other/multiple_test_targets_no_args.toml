args = ["test"]

status = "failed"

[bin]
name = "cargo"

[fs]
cwd = "../../../../fixtures/multiple_test_targets/package"
