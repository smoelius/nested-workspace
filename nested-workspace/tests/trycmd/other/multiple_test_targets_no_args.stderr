...
    Finished `test` profile [unoptimized + debuginfo] target(s) in [..]s
     Running unittests src/lib.rs (target/debug/deps/package-[..])
     Running tests/do_not_run.rs (target/debug/deps/do_not_run-[..])
error: test failed, to rerun pass `--test do_not_run`
